pom.xml

```diff
@@ -66,6 +65,13 @@
             <version>${htmlunit.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.vectomatic</groupId>
```

Comment:
Ohhh, I don't like that library, lol! (Had to fix it in the past for another client :D)